index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1 id="title">Our <strong>Survey</strong> Form</h1>
+
+      <!-- <section id="hero">
+        <div class="hero container">
+          <div>
+            <h1>Hello, <span></span></h1>
+            <h1>My Name is <span></span></h1>
+            <h1>Hamza <span></span></h1>
+            <a href="#" type="button" class="cta">Portfolio</a>
+          </div>
+        </div>
+      </section> -->
```

Comment:
Don't use comments. They will make your project look messy.

Reply:
I think your comment should be more polite. So it should also be a suggestion instead of making it look like an order.  For example, you can say, "**_Kindly_** try to don't use comments in your code. They will make your project look messy"